editor/components/button/style.scss

```diff
@@ -1,8 +1,14 @@
 .editor-button {
 	background: none;
 	border: none;
+	outline: none;
 
 	&:disabled {
 		opacity: 0.6;
 	}
+
+	&.is-active {
```

Comment:
Do we expect this to be the same styling we'll want for the [`:active` pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:active) ?

If so, should we add it here?
If not, should we be worried about confusion between the two unique states sharing the same name?